config/README.md

```diff
@@ -1,135 +1,169 @@
 
+Download & install the following tools to your workstation:
 
-## Creating a new Kubernetes cluster
+*   The [helm client](https://helm.sh/docs/using_helm/#installing-the-helm-client)
+    *   Required if deploying using the defined Kubernetes templates.
+*   kubectl
+    *   Required if deploying with Kubernetes.
+*   docker
+    *   Required if building new images of the DSS.
+*   Cockroachdb
+    *   Required to generate new CockroachDB certificates.
+*   Google Cloud SDK (if deploying on GCP)
+    * Required if deploying to Google Cloud.
+*   Golang.
+    *   Required if developing the DSS codebase.
 
-You need to create a Kubernetes cluster to run the cockroachdb instance and the
-gRPC and HTTP servers.  You can do this on any supported
-[cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
-or even on your own infrastructure.  Instructions for GCP are given below,
-consult the Kubernetes documentation for other providers.
 
-### Google Container Engine
+## Docker images
 
-Create a new cluster in the given zone:
 
-    gcloud --project $CLOUD_PROJECT container clusters create $CLUSTER_NAME --zone $ZONE
+The grpc-backend and http-gateway are the 2 main binaries for processing DSS 
+requests. These are both built and pushed to a docker registry of your choice.
+You can easily find out how to push to a docker registry through a quick search.
+All major cloud providers have a docker registry service, or you can set up your
+own.
 
-Fetch credentials for the cluster.  This populates your \~/.kube/config file
-and makes all future kubecfg commands target this cluster.
 
-    gcloud --project $CLOUD_PROJECT container clusters get-credentials $CLUSTER_NAME --zone $ZONE
+### Building Docker images
 
 
-## Creating a new cockroachdb cluster
+Set the environment variable `DOCKER_URL` to your docker registry url endpoint.
+
+Use the `build.sh` script in this directory to build and push an image tagged
+with the current date and git commit hash.
 
-1.  Create 5 static IP addresses.  How you do this depends on your cloud
-    provider.  3 IPs will be used for the 3 individual cockroachdb nodes, 1 IP
-    will be used to load balance amongst all the cockroachdb nodes, and 1 IP
-    will be used for the HTTPS frontend.  The HTTPS frontend IP will be used on
-    a Kubernetes Ingress, and if you're using Google Cloud it needs to be
-    created as a "Global" IP address.
+
+## Deploying the DSS on Kubernetes
+
+This secion discusses deploying a Kubernetes service, although you can deploy 
+a DSS node however you like as long as it meets the CockroachDB requirements
+above. You can do this on any supported
+[cloud provider](https://kubernetes.io/docs/concepts/cluster-administration/cloud-providers/)
+or even on your own infrastructure.  Consult the Kubernetes documentation for
+which ever provider you choose.
+
+1.  Create a new Kubernetes cluster as mentioned above. We recommend a new
```

Comment:
Where above is this mentioned?  It seems like original lines 45-62 would be useful to include, but are not actually included currently.

Reply:
The link directly above is for creating a kubernetes cluster